core/unix/coredump.c

```diff
@@ -484,11 +484,17 @@ os_dump_core_internal(dcontext_t *dcontext)
     // of LOAD program header (number of sections minus one since shstrtab does not
     // require a LOAD header).
     const ELF_OFF program_header_count = section_count;
+    ELF_OFF file_offset = sizeof(ELF_HEADER_TYPE) +
```

Comment:
Took me some time to figure out what this is the offset for: how about using a more descriptive variable name like "section_header_table_file_offset" which makes it clear.

Reply:
I've renamed it to core_file_offset. This variable points to where the content is located inside the core file and it's being updated when new header is added.